recipes/morph-kgc/meta.yaml

```diff
@@ -0,0 +1,182 @@
+{% set version = "2.8.0" %}
+
+# handle undefined PYTHON in `noarch: generic` outputs
+{% if PYTHON is not defined %}{% set PYTHON = "$PYTHON" %}{% endif %}
+
+package:
+  name: morph-kgc
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/m/morph-kgc/morph_kgc-{{ version }}.tar.gz
+  sha256: 1d95eeaa01fa9a272444e0d18f04151ff2be769bf9c816228fcb98f3f4e35b3c
+  patches:
+    - 0000-loosen-ruamel-yaml.patch
+
+build:
+  noarch: python
+  script:
+    - {{ PYTHON }} {{ RECIPE_DIR }}/check_recipe.py
```

Comment:
why do we have this please?

Reply:
The upstream has a bunch of `[extras]`. They have tight pins. `pip check` won't find changes, and wouldn't know to maps to the proposed `conda` outputs, where we can't use the same syntax.

If `meta.yaml` (or `recipe.yaml`) supported automatic verification of that somehow at the individual output level, that would help a lot, but I'm somewhat disinclined to remove things that would help bot-proposed PRs give me actionable information (and not ship bad versions)...